packages/services/src/services/Locale/Locale.js

```diff
@@ -85,20 +87,24 @@ class LocaleAPI {
    *    const list = await LocaleAPI.getList();
    * }
    */
-  static async getList(cc, lc) {
+  static async getList({ cc, lc }) {
     const url = `${_endpoint}/${cc}${lc}-utf8.json`;
 
     /**
      * if the json file for the cc-lc combo does not exist,
      * browser will automatically redirect to the us-en country list
      */
-    return await axios
+    const list = await axios
       .get(url, {
         headers: {
           'Content-Type': 'application/json; charset=utf-8',
         },
       })
       .then(response => response.data);
+
+    localStorage.setItem('countryList', JSON.stringify(list));
```

Comment:
@annawen1 wondering if we should either set this to expire, or consider using sessionStorage? Also, the country list doesn't change often, but we'd need a way to possibly cache bust it if necessary as well.

Reply:
yes, I'll need to do that (for the `ipcInfo` cookie as well!) - I believe NorthStar has both set with an expiration of a day...

Reply:
looking at v18, it seems that they set the expiration for the ipcInfo cookie to a year

var allowedTtl = IBMCore.www.module.truste.getUiAllowedStorageTtl(365 * 24 * 3600); in util.setIpcCookie() in misc.js

and then the country list in local storage to a day

var allowedTtl = IBMCore.www.module.truste.getUiAllowedStorageTtl(24 * 3600);

		// Cache translations locally for case of meta change page locale.
		selectorData = serviceData;

		// Store in global data in case page wants to use it.
		IBM.common.util.data.provide("localeSelector", serviceData);

		// Cache in local storage for 24 hours for subsequent page views.
		IBM.common.util.storage.setItem("v18lsdata", selectorData, allowedTtl);

Reply:
I set the cookie to a year and switched to session storage for the country list